### PR TITLE
devcert integration

### PR DESCRIPTION
**Pull request by asachin96**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
     "source-map-loader": "^0.2.4",
     "ts-loader": "^5.3.1",
     "typescript": "^3.3.3",
-    "webpack": "^4.29.5",
+    "webpack": "^4.29.6",
```
Comment (akrantz) [inline package.json]: Was an update needed for webpack?

File: webpack.config.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
+const devCerts = require("office-addin-dev-certs");
 const CleanWebpackPlugin = require("clean-webpack-plugin");
 const CopyWebpackPlugin = require("copy-webpack-plugin");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 const fs = require("fs");
 const webpack = require("webpack");
 
-module.exports = (env, options) => {
+module.exports = async (env, options) => {
+  const httsServerInfo = await devCerts.gethttpsServerOptions();
```
Comment (akrantz) [inline webpack.config.js]: In https://github.com/OfficeDev/Office-Addin-Scripts, this function name should be updated to have an uppercase h so it is `getHttpsServerOptions()`.

Comment (akrantz) [reply webpack.config.js]: You should take care to review changes so things like misspellings are caught. In this case, I would rename the variable to `https` which will match the object name.

**Comment by akrantz** [comment]
Now that office-addin-dev-certs is published, you can add it as a dev dependency to the package.

File: package.json (hunk 1 above)
Comment (asachin96) [reply package.json]: npm command might have updated this

**Pull request merged**